src/__experimental__/components/grouped-character/grouped-character.stories.mdx

```diff
@@ -0,0 +1,308 @@
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+
+import GroupedCharacter from ".";
+import { OriginalTextbox } from "../textbox";
+import Button from "../../../components/button";
+import Box from "../../../components/box";
+
+<Meta
+  title="Experimental/GroupedCharacter"
+  parameters={{ info: { disable: true } }}
+/>
+
+# GroupedCharacter
+
+Capture data with punctuation inside the field.
+
+Fields can have punctuation within them (most commonly characters like - or /). This gives the user a clue about the format accepted.
+
+## Quick Start
+
+```jsx
+import GroupedCharacter from "carbon-react/lib/__experimental__/components/grouped-character";
+```
+
+### Basic
+
+<Preview>
+  <Story name="Basic">
```

Comment:
could we rename all stories and start from nonCapital letter like `basic` instead of `Basic`